react-twitter-save-data-loading(hook)/src/App.js

```diff
@@ -23,7 +23,7 @@ import Tweet from './components/Twitter/Tweet';
 import Navbar from './components/Navbar/Navbar';
 import tweets from './data/tweets';
 import { IMAGE_TYPE } from './config';
-import { checkMobile } from './utils/helpers';
+import { checkIsSmallViewport } from './utils/helpers';
```

Comment:
Noting feedback from chat: the existing `checkMobile` test was relying on a combination of UA strings in order to determine if we were on a mobile or desktop, however, as we know such strings are fragile and cannot be relied on when browsers often fake UA strings. For now we've settled on a solution that relies on matchMedia for checks.